controller/lib/core/controller.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONTROLLER_H_
+#define CONTROLLER_H_
+
+#include "actuators.h"
+#include "blower_fsm.h"
+#include "network_protocol.pb.h"
+#include "pid.h"
+#include "units.h"
+
+// This class is here to allow integration of our controller into Modelica
+// software and run closed-loop tests in a simulated physical environment
+class Controller {
+public:
+  Controller();
+
+  ActuatorsState Run(Time now, const VentParams &params,
+                     const SensorReadings &readings);
+
+private:
+  float ComputeFanPower(Time now, const BlowerSystemState &desired_state,
```

Comment:
WDYT about following Google's ordering of (IIRC)

1. static constants
2. static member functions
3. non-static member functions
4. member variables.

Reply:
The constants didn't have to be here at all - moved them to .cpp.